scripts/scr_initialize_custom/scr_initialize_custom.gml

```diff
@@ -211,6 +211,16 @@ function scr_initialize_custom() {
 			strike_cruisers=8;
 	        gladius=7;
 	        hunters=3;
+			
+			if (global.chapter_name="Salamanders")
```

Comment:
probably use a switch statement here

Reply:
I'm still very new to coding so I'm not sure what a "switch statement" is but if you show me how I can implement it in the future.

Reply:
https://manual.yoyogames.com/GameMaker_Language/GML_Overview/Language_Features/switch.htm

but if you don't feel comfortable with GML yet, then just leave the ifs, it's fine here